src/framework/components/element/image-element.js

```diff
@@ -285,17 +403,54 @@ pc.extend(pc, function () {
             } else {
                 this._rect.set(value[0], value[1], value[2], value[3]);
             }
-            if (this._mesh) this._updateMesh(this._mesh);
+
+            if (this._mesh) {
+                this._updateMesh(this._mesh);
+            }
         }
     });
 
+    /**
+    * @name pc.ImageElement#border
+    * @type pc.Vec4
+    * @description The borders' size in pixels. The areas falling into the border region (where {@link pc.Vec4} is used to specify
+    * left, bottom, right and top border sizes) will be drawn 1:1 onto the elements' surface, allowing for the 9 patch buttons
+    * and other interface elements to stretch nicely.
+    * @example
+    * // make 10 pixel band around the texture be fixed.
+    * var element = this.entity.element;
+    * element.border = new pc.Vec4(10, 10, 10, 10);
+    */
+    Object.defineProperty(ImageElement.prototype, "border", {
+        get: function () {
+            return this._border;
+        },
+
+        set: function (value) {
+            if (value instanceof pc.Vec4) {
```

Comment:
It's avoidable – just the way it was done previously, so I retained it. If you agree, I would rather remove it as well.